### PR TITLE
Add _init_meta to PHACountsSpectrum

### PR DESCRIPTION
**Pull request by joleroi**
The meta info in ``SpectrumExtraction`` was taken from the ``obs_list`` instead of the header of the ``EVENTS`` extension

OUTDATED, see below

### CODE REVIEW

**Comment by cdeil** [comment]
I commented in https://github.com/gammapy/gammapy/issues/1348#issuecomment-376522311 to leave the discussion in one place.

**Comment by joleroi** [comment]
I changed this PR according to the discussion in #1348. Maybe a simple check if ``meta['EXPOSURE']`` has a unit is fine.

**Comment by cdeil** [comment]
The one build fail is a timeout. Merging now.

File: gammapy/data/data_store.py [hunk 1]
```diff
@@ -607,7 +607,10 @@ def observation_live_time_duration(self):
         Computed as ``t_live = t_observation * (1 - f_dead)``
         where ``f_dead`` is the dead-time fraction.
         """
-        return Quantity(self.obs_info['LIVETIME'], 'second')
+        if not isinstance(self.obs_info, Quantity):
```
[the remark this answers is not in the document]
Comment (joleroi) [reply gammapy/data/data_store.py]: :+1:  (Although I find this behaviour of ``Quantity`` a little weird)

**Pull request merged**